controllers.js

```diff
@@ -0,0 +1,11 @@
+const { selectTopics } = require("./models");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics(req.query)
```

Comment:
You shouldn't need to invoke selectTopics with anything at this stage as you are getting all of the topics